src/activity-vertical-and-horizontal-translations.xml

```diff
@@ -81,16 +79,14 @@
             </statement>
         </webwork>
     </exercise>
-
+<todo>This is not the best way to code this for formatting.</todo>
```

Comment:
If you change the `<md>` to `<md alignment="gather">` and leave out the `\amp` signs, I think that will do what you want. The = signs won't be aligned, but I don't think you need that here.